pbxplore/test/test_regression.py

```diff
@@ -240,15 +251,10 @@ def test_xtc_input(self):
         status = exe.wait()
         print(out.decode('utf-8'))
         print(err.decode('utf-8'))
-        if IS_MDANALYSIS:
-            # MDanalysis is available, PBassign should run and produce the
-            # correct output
-            assert status == 0, 'PBassign exited with an error'
-            _assert_identical_files(os.path.join(REFDIR, output_fname),
-                                    os.path.join(out_run_dir, output_fname))
-        else:
-            # MDanalysis is not available, PBassign should fail
-            assert status != 0, 'PBassign shoud not have exited with a 0 code'
```

Comment:
The new tests does not make sure that PBxplore behave as expected if MDAnalysis is missing.